src/qt/dbb_gui.cpp

```diff
@@ -1691,6 +1694,11 @@ void DBBDaemonGui::parseResponse(const UniValue& response, dbb_cmd_execution_sta
                             }, tr("Unlock the bootloader to install a new firmware"));
                             passwordAccepted();
                             return;
+                        } else {
+                            DBB::LogPrint("Outdated firmware\n", "");
```

Comment:
Should we raise a UI alert here?
Can be done later I guess.

Reply:
In that situation, the app screen goes to the 'device not connected' state, which I'd say works good enough as a UI alert.